docs/03-Linear-Methods-for-Regression/3.5-Methods-Using-Derived-Input-Directions.md

```diff
@@ -109,4 +109,4 @@ $$
 
 [^1]: Stone, M. and Brooks, R. J. (1990). Continuum regression: cross-validated sequentially constructed prediction embracing ordinary least squares, partial least squares and principal components regression (Corr: V54 p906-907), Journal of the Royal Statistical Society, Series B 52: 237–269.
 [^2]: Frank, I. and Friedman, J. (1993). A statistical view of some chemometrics
-regression tools (with discussion), Technometrics 35(2): 109–148.
```

Comment:
this seems to be just an end of a line change, which is unnecessary. But it helps me find that L112 should not be a new line.